README.md

```diff
@@ -8,7 +8,7 @@ The S2C2F SIG is a group working within the OpenSSF's Supply Chain Integrity Wor
 ## 
 **Motivation**
 
-[Background / use cases of the problem to be solved]
+OSS has become a critical aspect of any software supply chain. The S2C2F was designed based on known threats (i.e. tactics and techniques) used by adversaries to compromise OSS packages. By leveraging the framework, organizations can securely consume OSS dependencies into the developer's workflow and enhance their OSS governance program to address threats specific to OSS consumption.
```

Comment:
Can you please update the 2nd sentence to please read as: "By leveraging the framework, software development teams and organizations can securely consume...."